src/main/java/forestry/farming/logic/farmables/FarmableGE.java

```diff
@@ -22,17 +28,35 @@
 import forestry.api.arboriculture.TreeManager;
 import forestry.api.farming.ICrop;
 import forestry.api.farming.IFarmable;
+import forestry.api.genetics.AlleleManager;
 import forestry.arboriculture.ModuleArboriculture;
+import forestry.core.utils.datastructures.ItemStackMap;
+import forestry.core.utils.datastructures.StackMap;
 import forestry.farming.logic.crops.CropDestroy;
 
 public class FarmableGE implements IFarmable {
 
+	//StackMap used because a normal HashSet didn't seem to work
+	//TODO use items instead and normal set if this is still used in 1.14+ since items are flattened
+	private final StackMap<ItemStack, Boolean> windfall = new ItemStackMap<>();
```

Comment:
Try using `Collections.newSetFromMap(new ItemStackMap<>())`

Reply:
Thanks, I didn't know that existed.